docs/content/docs/data-structures/sequences.md

```diff
@@ -1,402 +0,0 @@
----
```

Comment:
Is this content still useful? Should we move it elsewhere or mark it as a draft? @Abe27342 do you have any opinion on this?

Reply:
FWIW, most of the content in this deleted page came from the sequence README, and most of that is also on the string page. That said, I should add a redirect from the old page so URLs don't break.

`AUTO-GENERATED-CONTENT:START (INCLUDE:path=packages/dds/sequence/README.md&start=2&end=377)`

Reply:
Hmmm, I may have jumped the gun here... I'll bring this one back.

Reply:
The content is still useful and should appear *somewhere*, but if I were starting from scratch I'd put it all in the SharedString documentation. Non-shared-string sequences don't have a great answer right now (SharedTree should hopefully make this better), so having two separate pages here is a bit misleading.

But I'm ok with the status quo of 2 pages.